PPI/manual_work/Jul_2023/manual_tables.sql

```diff
@@ -0,0 +1,383 @@
+-- concept_manual backup
+DO
+$body$
+    DECLARE
+        update text;
+    BEGIN
+        SELECT TO_CHAR(CURRENT_DATE, 'YYYY_MM_DD')
+        INTO update;
+        EXECUTE format('create table %I as select * from concept_manual',
+                       'concept_manual_backup_' || update);
+    END
+$body$;
+
+SELECT * FROM dev_ppi.concept_manual_backup_2023_07_25;
+
+-- concept_relationship_manual backup
+DO
+$body$
+    DECLARE
+        update text;
+    BEGIN
+        SELECT TO_CHAR(CURRENT_DATE, 'YYYY_MM_DD')
+        INTO update;
+        EXECUTE format('create table %I as select * from concept_relationship_manual',
+                       'concept_relationship_manual_backup_' || update);
+    END
+$body$;
+
+SELECT * FROM dev_ppi.concept_relationship_manual_backup_2023_07_25;
+
+TRUNCATE concept_manual;
+TRUNCATE concept_relationship_manual;
+
+-- deprecate previous Mental Health and Well-Being Module concepts --229
```

Comment:
It is a prominent example of bad coding. Collect these concepts in some kind of manual table and insert them into concept_manual from there. How did you assess that every concept from the module has been deleted?

Reply:
done